subsys/bluetooth/controller/ll_sw/ull_adv_aux.c

```diff
@@ -1060,12 +1116,13 @@ void ull_adv_aux_offset_get(struct ll_adv_set *adv)
 	LL_ASSERT(!ret);
 }
 
-struct pdu_adv_aux_ptr *ull_adv_aux_lll_offset_fill(uint32_t ticks_offset,
-						    uint32_t start_us,
-						    struct pdu_adv *pdu)
+struct pdu_adv_aux_ptr *ull_adv_aux_lll_offset_fill(struct lll_adv_aux *lll_aux,
```

Comment:
`lll_aux` looks like it is unused in this function.

Reply:
Good catch.